### PR TITLE
[2.7.11] Support k8s 1.27 for system-upgrade-controller

### PR DESCRIPTION
**Pull request by thatmidwesterncoder**
Issue: https://github.com/rancher/rancher/issues/44114

Pulling the changes to support k8s v1.27 from https://github.com/rancher/system-charts/pull/594

### CODE REVIEW

**Review by HarrisonWAffel — APPROVED** [review]

File: charts/rancher-k3s-upgrader/0.6.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: rancher-k3s-upgrader
+description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+  Users do not need to manually upgrade this app. It will be automatically upgraded to the latest version when upgrading a cluster.
+home: https://github.com/rancher/system-charts/blob/dev-v2.7.11/charts/rancher-k3s-upgrader
+sources:
+  - "https://github.com/rancher/system-charts/blob/dev-v2.7.11/charts/rancher-k3s-upgrader"
```
Comment (jiaqiluo) [inline charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:7]: Nit:  What will be the ultimate branch for hosting this chart, will it be `release-v2.7`? 
How did we handle it in older versions of this chart?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]
I think my above question should not be a blocker for merging the PR.

File: charts/rancher-k3s-upgrader/0.6.0/Chart.yaml (hunk 1 above)
Comment (thatmidwesterncoder) [reply charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:7]: So the chart version already exists in `dev-v2.8` and above but with different rancher requirements - will the 2.7.11 branch get merged back into 2.7? If so I'm happy to update it to just `release-v2.7`.

Comment (jiaqiluo) [reply charts/rancher-k3s-upgrader/0.6.0/Chart.yaml:7]: I think about it again, I think the branch `dev-v2.7.11` will be there forever as a reference to the source, so it should be ok to keep the file as it is.

**Pull request merged**